packages/compiler/src/experimental/typekit/kits/type.ts

```diff
@@ -214,7 +214,7 @@ defineKit<TypekitExtension>({
       return clone;
     },
     isError(type) {
-      return isErrorType(type);
+      return isErrorType(type) || isErrorModel(this.program, type);
```

Comment:
I kinda feel like it should never have been `isErrorType`.

This is not a type you would receive in your emitter or decorator unless there is a bug

Reply:
the doc of the typekit also just says it returns true for model with `@error`

Reply:
which on a side note should escape the `@error` in backtick or it will be treated as a jsdoc tag

Reply:
Gotcha, so you're saying that we should not be checking isErrorType at all, _only_ isErrorModel? 

The issue is not very descriptive so I had to read between the lines a bit. It says "The $.type.isError typekit documentation claims it checks for the https://github.com/error decorator, which would be useful, but it unfortunately **just** checks if the type is the built-in error type (which should be exposed via built-ins)."

So it's a bit hard to tell _what_ exactly is the bug. Are you saying that the correct implementation would just be `return isErrorModel(this.program, type)` ?

Reply:
yeah I think so